the ar_EG.ISO8859-6 does not have RIGHT-TO-LEFT character which is used in Egyptian currency symbol.

--- cldr_core.orig//common/supplemental/characters.xml
+++ cldr_core/common/supplemental/characters.xml
@@ -529,6 +529,9 @@
 			<character value = "ï¿¦">
 				<substitute>KRW</substitute>
 			</character>
+			<character value = "&#x200f;">
+				<substitute></substitute>
+			</character>
 		</character-fallback> 
 	</characters>
 </supplementalData>
